analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -596,6 +613,39 @@ def construct_config_handler(cls, args):
             cmdline_checkers,
             'enable_all' in args and args.enable_all)
 
+        # If --stats flag is provided then enable statistics-based checkers
+        # unless explicitly disabled by the user.
+        if 'stats_enabled' in args:
+            # In some Clang versions these checkers may be in some checker
+            # group (e.g. alpha...), so let's find the actual checker name
+            # first.
```

Comment:
I see what you wanted to say, but its a little sloppy. How about this:

"In some Clang versions, statisticsbased checkers are in the alpha package (alpha.statisticsbased), and maybe not in others. Lets figure out the actual checker name first"